src/blocks/sound.rs

```diff
@@ -653,6 +663,10 @@ impl SoundDevice for PulseAudioSoundDevice {
         self.active_port.clone()
```

Comment:
Not part of your PR, but can you please remove this `clone()` too?

Reply:
Ditto